examples/getting_started/pendulum.py

```diff
@@ -13,6 +16,23 @@
 )
 
 
+def custom_dynamic(states, controls, parameters, nlp):
+    Dynamics.apply_parameters(parameters, nlp)
+    q, qdot, tau = Dynamics.dispatch_q_qdot_tau_data(states, controls, nlp)
+
+    qdot_reduced = nlp["q_mapping"].reduce.map(qdot)
+    qddot = biorbd.Model.ForwardDynamics(nlp["model"], q, qdot, tau).to_mx()
+    qddot_reduced = nlp["q_dot_mapping"].reduce.map(qddot)
```

Comment:
As a matter of simplicity for the user, I would not use reduce mapping in the example. 
(Also, the example should not be for pendulum, but in another file called something like custom_dynamics)